objects/_objects.ratio.scss

```diff
@@ -83,3 +83,26 @@ $inuit-ratios: (
 }
 
 /* stylelint-enable */
+
+
+
+
+
```

Comment:
Do you need this many line breaks? Would be nice to keep it consistent and have only 3 (rather than 5).

Reply:
@olivertappin Actually, 5 empty lines above new sections in a file is a consistent convention across the framework. See [media object](https://github.com/inuitcss/inuitcss/blob/29ec23cfdfc5e8004f31e41a4c9f701cfa0ba642/objects/_objects.media.scss) for example.